next.config.js

```diff
@@ -9,27 +9,32 @@ module.exports = {
         hostname: process.env.NEXT_IMAGE_DOMAIN,
       },
     ],
+    deviceSizes: [
```

Comment:
Can you tell me why you chose these breakpoints?

Reply:
Mostly arbitrary. I took the default values and commented out the ones I didn't want. I tried to get something close-ish to the breakpoints.

Reply:
Why not align them with the breakpoints of the design system?

Reply:
meh, it's not really necessary IMO. Images don't need to break down at the same breakpoint. Just as long as the image is larger than the area it's in. Almost all images use a css cover property anyways

Reply:
This seems like a thing to ask what others on the team are doing and how they set up their responsive images. It is in the 'doesn't really matter' category but this seems like a good opportunity to engage and align with others.